comic_dl/sites/manganelo.py

```diff
@@ -63,8 +71,21 @@ def single_chapter(self, manga_url, download_directory, conversion, keep_files):
             links.append(image_link)
             i += 1
 
-        globalFunctions.GlobalFunctions().multithread_download(chapter_number, comic_name, manga_url, directory_path,
-                                                               file_names, links, self.logging)
+        try:
+            globalFunctions.GlobalFunctions().multithread_download(chapter_number, comic_name, manga_url,
+                                                                   directory_path,
+                                                                   file_names, links, self.logging)
+        except Exception as ex:
+            # try mirror server with a referer = origin page
+            if referer is None:
```

Comment:
```
if not referer
```
this will cover accidental false/true values as well.

Reply:
done